.lando.yml

```diff
@@ -56,11 +60,6 @@ services:
         - seccomp:unconfined
       command: ["/usr/local/bin/supervisord", "-c", "/etc/supervisord.conf"]
 
-events:
-  post-start:
-    - node: /bin/sh -c "chmod -R 775 /var/www"
-    - node: echo "Building theme asset files..."
-    - node: cd /app/web/themes/custom/cloudy && yarn run build
```

Comment:
I'm okay with removing the default build, but we need to update our README to explain to the developers that they don't actually have an accurate theme until they've run lando refresh—which shouldn't be a huge burden as they need to run lando refresh to get the latest changes from master for the new branch they are working on. I nearly always to a lando latest and lando refresh after a rebuild anyway.

Reply:
On the first boot up, I don't think they would have any theme. I have put back this whole block.